src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -241,8 +212,8 @@ def step(self, t, t_new, current_rv):
         local_errors = self._estimate_local_error(
             pred_rv,
             t_new,
-            self.sigma_squared_mle * proc_noise_cov,
-            np.sqrt(self.sigma_squared_mle) * proc_noise_cov_cholesky,
+            local_squared_diffusion * proc_noise_cov,
```

Comment:
So even in the fixed diffusion model, we use just the diffusion estimate from this step, not the full MSE? That would be a different behaviour to the way ProbNumDiffEq.jl treats the fixed diffusion.

Reply:
Hmmm yes I see the point. Personally, I think this is a better option in the sense of not breaking when ODEs get more challenging, etc.. But you're right in the sense that we should offer all options. I will ping you offline :)